pbrEngine/engine.py

```diff
@@ -198,6 +198,8 @@ def _crash(reason=None):
 
         self._matchVolume = 100
         self._fMatchAnnouncer = True
+        self.musicEnabled = False
+        self._musicCurrentlyEnabled = False
```

Comment:
how are these two variables used? what two of them?

Reply:
musicEnabled is set by the core to signify whether music should be enabled every match, _musicCurrentlyEnabled is whether the current match has music